README.md

```diff
@@ -1,23 +1,27 @@
 # Calico Networking for rkt
 
 ## Building the plugin
-To build the calico-kubernetes plugin, clone this repository and run `make`.  This will build the binary, as well as run the unit tests.  To just build the binary, with no tests, run `make binary`.  To only run the unit tests, simply run `make ut`.
+
+To build the Calico Networking Plugin for Rkt, clone this repository and run `make`.  This will build the binary, as well as run the unit tests.  To just build the binary, with no tests, run `make binary`.  To only run the unit tests, simply run `make ut`.
```

Comment:
I would say something here about building the plugin locally and also point at the releases page to show that binaries are available and it is not required to build locally.